### DISCUSSION TITLE
Possibility to listen for "open"/"value" change event on ui.expansion

### DISCUSSION THREAD

**Discussion by aersam** [zauberzeug/nicegui#1354] [Ideas / Feature Requests]
I tried to add a change listener to ui.expansion. Neither of those does work:

```python
with ui.expansion("somthing", on_value_change=_on_change) as exp:
    pass
```

```python
with ui.expansion("somthing") as exp:
     exp.on_value_change(_on_change)
```

Is there another way to do this?

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @aersam! The `on_value_change` parameter was indeed missing.
I fixed it in PR #1356 which will be released shortly.
